development_guidance/development_teams.md

```diff
@@ -0,0 +1,53 @@
+# Development Teams
+
+This guide contains various concepts which form a set of expectations we have on how our development teams will operate.
+
+## Full Stack, Vertical Slices
+
+We aim to create teams of full stack developers, rather than creating teams that specialise in disciplines like front end development, testing, devops etc.
+
+We do this as we believe it allows us to create teams that excel at delivering rapid change (and therefore value) to users.
+
+In order to maximise the benefits of creating teams this way, we need to organise our work deliberately to play to these strengths -
+ - Work should sliced 'vertically' meaning a given task is implemented in the back end and front end, and tested
+ - Confident testing is part of the acceptance criteria for a given task
+ - Tasks are refined to the point where any member of the development team could action them, from new features to infrastructure changes to bugfixes
+ - Teams should regularly practice pairing and actively work to avoid knowledge silos
+ - Work should have a clear path to production and deploy as often and quickly as possible
+
+## Stacks and Tools
+
+Where we mention we 'prefer' to use certain tools or packages, teams are free to deviate if it is deemed necessary, for other tools or stacks we consider this section mandatory.
+
+We develop all new digital services as single repository .NET Core (3.1 LTS) MVC applications, using Razor for templating. .NET Core 5+ is acceptable, but we opt to use the long term supported version if possible.
```

Comment:
Is it worth not specifying a version number here? If we specify that we aim to use the latest LTS, but latest stable is acceptable, would that work? Will save on having to keep this document up-to-date every time .NET Core updates.